packages/apps/docs/package.json

```diff
@@ -11,12 +11,12 @@
     "7d:query": "npx 7d",
     "algolia-dev": "ALGOLIA_INDEX_NAME=docs_website_dev OPENAI_API_KEY=dummy-key121 npx 7d ingest --files 'src/pages/**/*.md' --files 'src/pages/**/*.mdx' --files 'src/specs/**/*.json' --db algolia --skip-embeddings",
     "algolia-prod": "ALGOLIA_INDEX_NAME=docs_website_prod OPENAI_API_KEY=dummy-key121 npx 7d ingest --files 'src/pages/**/*.md' --files 'src/pages/**/*.mdx'  --files 'src/specs/**/*.json' --db algolia --skip-embeddings",
-    "build": "NODE_OPTIONS=--max-old-space-size=8192 pnpm run build:scripts && next build",
+    "build": "NODE_OPTIONS=--max-old-space-size=9216 pnpm run build:scripts && next build",
```

Comment:
Vercel build machines have a max of 8192MB of ram available for a single build, upgrading this shouldn't do much. Given that we're probably going to refactor this very soon anyway: Leave it for now